Assets/MixedRealityToolkit/Services/MixedRealityToolkit.cs

```diff
@@ -672,7 +606,7 @@ public static void AssertIsInitialized()
         /// <summary>
         /// Returns whether the instance has been initialized or not.
         /// </summary>
-        public static bool IsInitialized => instance != null;
+        public static bool IsInitialized => Instance != null;
```

Comment:
This is a pretty big breaking change that has wide reaching implications.

Reply:
Yep. Hence the review request. As it stands support scene switching is completely broken.

Reply:
With the old behavior this would also never return false, as the Instance getter would create the instance. I'm against making this change as well.